docs/product/data-management-settings/event-grouping/index.mdx

```diff
@@ -8,36 +8,27 @@ A fingerprint is a way to uniquely identify an event, and all events have one. E
 
-<Note>
+When you create a Sentry project, the most recent version of the error grouping algorithm is automatically selected. This ensures that grouping behavior is consistent within a project.
 
-Stack trace rules can work as a combination of both SDK and project settings. As a result, we maintain the documentation in one location.
+To upgrade an existing project to a new error grouping algorithm version, navigate to **Settings > Projects > [project] > Issue Grouping > Upgrade Grouping**. After upgrading to a new error grouping algorithm, you will very likely see new groups being created.
 
-</Note>
+All versions consider the `fingerprint` first, the `stack trace` next, then the `exception`, and then finally the `message`.
 
-You can see a fingerprint by opening an issue, clicking the JSON link, and finding the _fingerprint_ property in that file. If the default grouping was used, you'll see "default" written there. If a different grouping was used, you'll see the actual fingerprint value itself.
+To see how an isssue was grouped, go to its **Issue Details** page and scroll down to "Event Grouping Information". There, you'll see if it was grouped by a fingerprint, stack trace, exception, or message.
 
-## Error Grouping Algorithms
+### Grouping by Built-In Fingerprinting Rules
 
-Each time default error grouping behavior is modified, Sentry releases it as a new version. As a result, modifications to the default behavior do not affect the grouping of existing issues.
+Some kinds of errors, like chunk load errors and hydration errors, often have various different error types, values, or transaction messsages that may result in separate but similar issues. To better handle these cases, Sentry maintains a set of built-in fingerprinting rules to better group such errors.
 
-When you create a Sentry project, the most recent version of the error grouping algorithm is automatically selected. This ensures that grouping behavior is consistent within a project.
+Issues grouped by Sentry's built-in fingerprinting rules will list "Sentry Defined Fingerprint" in the "Event Grouping Information" at the bottom of the **Issue Details** page.
 
-To upgrade an existing project to a new error grouping algorithm version, navigate to **Settings > Projects > [project] > Issue Grouping > Upgrade Grouping**. After upgrading to a new error grouping algorithm, you will very likely see new groups being created.
+![event grouping information for a sentry defined fingerprint](sentry-defined-fingerprint.png)
 
-All versions consider the `stack trace` first, the `exception` next, and then finally the `message`.
+Built-in fingerprinting rules work the same as custom fingerprinting rules and are applied by default. If a custom fingerprinting rule defined by a user matches an event, the custom rule will always override the built-in ones.
```

Comment:
IMO "override" isn't best word here, because custom ones would be applied firsts and it wouldn't even get to built-in ones. The way fingerprinting works is that it applies the _first_ rule that matches and that's it. 

```suggestion
Built-in fingerprinting rules work the same as custom fingerprinting rules and are applied by default. If an user defined custom fingerprinting rule matches an event, the custom rule will always take precedence over the built-in ones.
```